GuiFunctions/Databases/DownloadUniProtDatabaseFunctions.cs

```diff
@@ -1,4 +1,6 @@
-﻿using System.Text;
+﻿using Proteomics.ProteolyticDigestion;
+using System.Text;
+using static Org.BouncyCastle.Bcpg.Attr.ImageAttrib;
```

Comment:
what is this bouncy castle thing?

Reply:
It has probably been added automatically by visual studio, I see it is not used, I will delete it.